src/plugins/test_plugin.cpp

```diff
@@ -22,7 +22,7 @@ void TestPlugin::stop() {
 
 std::vector<std::pair<PluginFunctionName, PluginFunctionPointer>> TestPlugin::provided_user_executable_functions() {
   return {{"OurFreelyChoosableFunctionName", [&]() { this->a_user_executable_function(); }},
-          {"SpecialFunction17", [&]() { this->a_static_user_executable_function(); }}};
+          {"SpecialFunction17", [&]() { opossum::TestPlugin::a_static_user_executable_function(); }}};
```

Comment:
The opossum is retired. https://naturenibble.com/wp-content/uploads/2022/04/Types-of-Possums-05082022.jpg